runtime/help/commands.md

```diff
@@ -107,7 +107,11 @@ quotes here but these are not necessary when entering the command in micro.
 
 * `plugin available`: show available plugins that can be installed.
 
-* `reload`: reloads all runtime files.
+* `reload`: reloads all runtime files and reinitializes all configurations.
+   In case `micro` was started with a `-<option>` being set to something other
+   than the default value and other than set within the `settings.json`
+   then this volatile option will be reverted to the default or to the value
+   defined in `settings.json`.
```

Comment:
The `reload` description is expanded also in PR #3240 (still not merged, but IMHO it is ready for merge).
This change would make it even more lengthy and confusing.

...Now it actually seems a bit more reasonable to me if `reload` will not change volatile settings. Its job is to update those settings that were loaded from `settings.json` (since `settings.json` changed), arguably it should not update those settings that were not loaded from `settings.json` in the first place.

Reply:
> This change would make it even more lengthy and confusing.

Ok, drop it (locally) already.

> ...Now it actually seems a bit more reasonable to me if `reload` will not change volatile settings.

Yes, that's an other way around. Sounds feasible and leaves the changed volatiles alone.

Reply:
> Sounds feasible and leaves the changed volatiles alone.

At least as long as they aren't changed by the `settings.json` too.